### PR TITLE
New package: sov-0.73

### PR DESCRIPTION
**Pull request by brihadeesh**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/sov/template [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'sov'
+pkgname=sov
+version=0.63
+revision=1
+build_style=meson
+hostmakedepends="pkg-config cmake"
```
Comment (abenson) [inline srcpkgs/sov/template:6]: You need `wayland-devel` here as well

Comment (brihadeesh) [reply srcpkgs/sov/template:6]: In hostmakedepends?

Comment (brihadeesh) [reply srcpkgs/sov/template]: Alright this seems to fix the failed checks

**Comment by brihadeesh** [comment]
Hi @abenson @paper42 could either of you review this? (just saw that it's gone stale)

File: srcpkgs/sov/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'sov'
+pkgname=sov
+version=0.63
+revision=1
+build_style=meson
+hostmakedepends="pkg-config cmake wayland-devel"
```
Comment (paper42) [inline srcpkgs/sov/template:6]: cmake is an alternative to pkg-config in this case, drop it

File: srcpkgs/sov/template [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+makedepends="wayland-protocols wayland-devel freetype-devel"
+depends="wayland"
```
Comment (paper42) [inline srcpkgs/sov/template:8]: this should be autodetected, remove this line

Comment (brihadeesh) [reply srcpkgs/sov/template:8]: The `depends` assignment?

Comment (paper42) [reply srcpkgs/sov/template:8]: yes

**Comment by brihadeesh** [comment]
Not sure why my np_sov branch is so outdated or why it's diverging when I rebase with upstream here.

**Comment by brihadeesh** [comment]
Ok that should've fixed it.

**Comment by brihadeesh** [comment]
Hey, @paper42 @abenson gentle reminder to review this.

**Comment by abenson** [comment]
Update to the latest version and force push to this branch

**Comment by brihadeesh** [comment]
Hey, @abenson it's done. Somehow keep forgetting to rebase before pushing

**Comment by brihadeesh** [comment]
Hi, @abenson I've updated the template to reflect the 0.73 which is one version number behind the current release. The latest release i.e. 0.74 doesn't appear to build and I think I might have to get that fixed upstream.

**Pull request closed without merging**